prow/git/v2/remote_test.go

```diff
@@ -104,12 +104,12 @@ func usernameVendor(logins []stringWithError) func() (string, error) {
 }
 
 // tokenVendor returns a func that vends tokens from the provided slice, in order
-func tokenVendor(tokens [][]byte) func() []byte {
+func tokenVendor(tokens []string) func(org string) (string, error) {
```

Comment:
Would this be better named fakeTokenGetter?

The lingo surrounding these tokens is too varied across various files. I see "token", "token getter", "generator", and here "vendor". Would improve readability to consolidate.

Reply:
Sure, updated that.